stl/inc/xutility

```diff
@@ -3573,7 +3573,7 @@ namespace ranges {
         }
 
         constexpr subrange& advance(const iter_difference_t<_It> _Count) {
-            // Per LWG-3413, this has defined behavior when _Count < 0.
+            // Per [range.subrange.access], this has defined behavior when _Count < 0.

```

Comment:
Ditto missing document number.

This comment was here to explain why the behavior differed from the spec (poorly, given that the issue number was wrong). It now seems strange to have a comment that says "this behaves as the spec says it should". Strike it?

Reply:
Yeah, we can drop this comment (I wasn't sure if you still wanted it, so I erred on the side of retaining it).